README.md

```diff
@@ -506,6 +506,26 @@ extra conditions contradict the conditions in the rules, the Transfer can never
 The Receiver must post a comment describing the resolution of these extra conditions with validity as a result, before
 he can accept the Transfer.
 
+# Representation of Nomics
+
+## Mutable Rules
+
+**377** *Representation Theory*
+
+A **Nomic** is a game in which the rules may change. 
+
+A **Nomic game state** is a record of a Nomic, interpretable
+by humans, specifying a current rule-set and any other varying attributes. 
+
+A **Nomic representation** is a set of Nomic game states, one of which is
```

Comment:
I'm kind of split on doing "Many-Worlds". If it's many worlds then I'll define "master" to be the current state of a GH-repo representation. If we end up on one world, then I will indeed define "pimotte/master" as current state.